lib/main.dart

```diff
@@ -200,6 +201,16 @@ class _ObtainiumState extends State<Obtainium> {
                   context.locale.languageCode)) {
         settingsProvider.resetLocaleSafe(context);
       }
+      settingsProvider.addListener(() async {
```

Comment:
We shouldn't need to store `appFont` in `settingsProvider` since it isn't used anywhere outside of this main build function. We also shouldn't need to use  `settingsProvider.addListener` since we're already watching the `settingsProvider` so the build function is re-run when any setting changes.

I tested the following and it works (replaces current lines 204-213):
```dart
      if (settingsProvider.tryUseSystemFont && fontToUse == "Metropolis") {
        NativeFeatures.tryLoadSystemFont().then((fontLoaded) {
          if (fontLoaded) {
            setState(() {
              fontToUse = "SystemFont";
            });
          }
        });
      } else if (!settingsProvider.tryUseSystemFont &&
          fontToUse != "Metropolis") {
        setState(() {
          fontToUse = "Metropolis";
        });
      }
```

Then we can have `String fontToUse = 'Metropolis';` initialized at the top in `_ObtainiumState`, and can remove the `appFont` getter/setter from `settingsProvider`.